beacon-chain/sync/receive_block.go

```diff
@@ -65,21 +72,25 @@ func (rs *RegularSync) receiveBlock(msg p2p.Message) error {
 		return nil
```

Comment:
No warning or anything? Can we log something here?

Reply:
Can we add an attribute to the span to say the block was invalid?

Reply:
Added